media/server/ipc/source/SessionManagementServer.cpp

```diff
@@ -82,6 +88,56 @@ bool SessionManagementServer::initialize(const std::string &socketName, unsigned
     {
         RIALTO_SERVER_LOG_SYS_WARN(errno, "Failed to change the permissions on the IPC socket");
     }
+
+    const uid_t noOwnerChange = -1; // -1 means chown() won't change the owner
+    uid_t ownerId = noOwnerChange;
+    if (!socketOwner.empty())
+    {
+        errno = 0;
+        struct passwd passwordStruct;
+        struct passwd *passwordResult;
+        char stringBuffer[1024];
```

Comment:
magic number, add constexpr variable for buffer size and use variable for the bufsize in getpwnam_r